scalyr_agent/builtin_monitors/kubernetes_events_monitor.py

```diff
@@ -340,12 +321,11 @@ def _check_if_alive( self, k8s, node ):
 
         result = {}
         try:
-            options = ApiQueryOptions(max_retries=self.__max_query_retries, rate_limiter=self.__rate_limiter)
             # this call will throw an exception on failure
-            result = k8s.query_api_with_retries('/api/v1/nodes/%s' % node, options,
+            result = k8s.query_api_with_retries('/api/v1/nodes/%s' % node,
                                                 retry_error_context=node, retry_error_limit_key='k8se_check_if_alive')
         except Exception, e:
-            # TODO-163: add debug statement
+            global_log.log(scalyr_logging.DEBUG_LEVEL_1, "_check_if_alive False for node %s" % node)
```

Comment:
Can we print the exception here (not necessarily the traceback, but at least the exception message).